action.yml

```diff
@@ -2,16 +2,16 @@ name: 'MeshMap-Snapshot'
 description: 'Walks in application and takes a shot of your infrastructure using Meshery Extension MeshMap'
 author: Layer5
 inputs:
-  designId:  # id of input
+  designId:  # id of input  #deprecated
```

Comment:
@Abhishek-kumar09 please include an example input in each input - either in the description or as a default value.

For example, here, I can't tell if it's just the UUID or the absolute path to Meshery Cloud that is needed.

Reply:
I see `application_url:` below, which helps contrast the difference, however, an example would be helpful.

Let's not use the phrase "Give the" - delete that part, please.

Reply:
Done.